fs/nfs/Kconfig

```diff
@@ -17,6 +17,7 @@ config NFS_DONT_BIND_TCP_SOCKET
 	bool
 	default n
 	depends on NFS
```

Comment:
18    depends on NFS

Within

    13 if NFS

Is redundant, is it not?

Reply:
Seems to be true. Thanks for noticing!

Reply:
Fixed!